Library/Homebrew/extend/os/mac/missing_formula.rb

```diff
@@ -14,12 +14,16 @@ def blacklisted_reason(name)
         end
       end
 
-      def cask_reason(name, silent: false, show_info: false)
```

Comment:
I think this version is nicer rather than using `Homebrew.args`

Reply:
Sorry, I think I misunderstood your previous comment! I'll revert it.